servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -376,7 +376,9 @@ def format_scattered_task(task_name, task_metadata):
             if shard.get('start') and min_start > _parse_datetime(
                     shard.get('start')):
                 min_start = _parse_datetime(shard.get('start'))
-            if shard.get('executionStatus') not in ['Failed', 'Done']:
+            if shard.get('executionStatus') not in [
+                    'Failed', 'Done', 'Aborted'
```

Comment:
Don't know PEP, Is this two-indents (8-spaces) or a single indent (4-spaces)

Reply:
it's whatever the python linter Gods say it should be 🤷‍♀